spec/controllers/reservations_controller_spec.rb

```diff
@@ -6,28 +6,27 @@
   render_views
 
   before(:all) do
-    @app_config = FactoryGirl.create(:app_config)
-
     @user = FactoryGirl.create(:user)
     @banned = FactoryGirl.create(:banned)
     @checkout_person = FactoryGirl.create(:checkout_person)
     @admin = FactoryGirl.create(:admin)
   end
 
   after(:all) do
-    User.delete_all
-    AppConfig.delete_all
-    Reservation.delete_all
-    Category.delete_all
-    EquipmentModel.delete_all
```

Comment:
Do we think that this the problem before? The original issue was the presence of unexpected equipment in the database. Did `delete_all` mess up because of soft-deletion / `permanent_record`? (This is all pure conjecture)